bin/blackbox_view

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+#
+# blackbox_edit.sh -- Decrypt a file temporarily for edition, then re-encrypts it again
+#
+set -e
+[ -z $EDITOR ] && EDITOR="vi"
+blackbox_home=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source ${blackbox_home}/_blackbox_common.sh
+
+for param in """$@""" ; do
+  unencrypted_file=$(get_unencrypted_filename "$param")
+  if  ! is_on_cryptlist "$param" && ! is_on_cryptlist "$unencrypted_file" ; then
```

Comment:
blackbox_view wasn't encrypting any files...only decrypting and deleting.  But as mentioned above, this is unnecessary since blackbox_cat already exists.